### PR TITLE
Feat/orders service

### PR DESCRIPTION
**Pull request by gustavosf2003**

### CODE REVIEW

File: backend/utils/order.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function getCurrentTime() {
+  const date = new Date();
+  const day = date.getDate().toString().padStart(2, "0");
```
Comment (vitor-afonso) [inline backend/utils/order.js:3]: Smart use of padStart ;)

File: backend/email.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      html: html,
+    };
+
+    transporter.sendMail(info, (error, info) => {
+      if (error) {
+        console.log(error);
+      }
+    });
+  } catch (e) {
+    console.log(e);
```
Comment (vitor-afonso) [inline backend/email.js:40]: I would also return a message to be handled in the frontend and provide feedback to the user.
`return res.status(500).json({ message: "Something went wrong: " + e })`

File: backend/email.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    let info = {
+      from: "freshmarket.lisbon@gmail.com",
+      to: order.email,
+      subject: "Order on the way",
+      html: html,
+    };
+
+    transporter.sendMail(info, (error, info) => {
+      if (error) {
+        console.log(error);
```
Comment (vitor-afonso) [inline backend/email.js]: I would add `res` as prop of  the function, and return a message to be handled in the frontend and provide feedback to the user.
`return res.status(400).json({ message: "Something went wrong: " + e })`

File: backend/app.js [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+app.engine("handlebars", engine());
+app.set("view engine", "handlebars");
+app.set("views", "./views");
 
 app.use(express.json());
 
 app.post("/order", (req, res) => {
-  console.log(req.body);
+  let order = req.body;
+  sendOrderConfirmationEmail(order);
```
Comment (vitor-afonso) [inline backend/app.js:15]: Would add `res` as second parameter and handle the response inside the function

File: backend/views/order.handlebars [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  </head>
+  <body>
+    <div class="container">
+      <div class="header-image"></div>
+      <h1>Hey {{name}}</h1>
+      <p>
+        Thank you for choosing Fresh Market as your grocery store! We're excited
+        to let you know that your order has been confirmed and is on the way to
+        <i>
+          <u>{{address}}</u>
```
Comment (vitor-afonso) [inline backend/views/order.handlebars:30]: Why are you using tag `<u>`?

File: backend/app.js (hunk 4 above)
Comment (vitor-afonso) [inline backend/app.js]: Would add `res` as second parameter and handle the response inside the function

File: backend/email.js (hunk 2 above)
Comment (gustavosf2003) [reply backend/email.js:40]: Makes sense 👍

File: backend/views/order.handlebars (hunk 5 above)
Comment (gustavosf2003) [reply backend/views/order.handlebars:30]: To underline the text.

`The <u> tag in HTML is used to underline text. It is one of the many formatting tags available in HTML to enhance the presentation of content on a web page. The <u> tag is typically used to indicate a link, a keyword, or a phrase that needs emphasis or attention.`

**Pull request merged**